foundations/installations/installing_node.md

```diff
@@ -82,3 +87,112 @@ nvm use node
 Now when you run `node -v` you should see `v14.10.0` or something similar.
 
 If you see that, you have successfully installed node!
+
+
+# Installation on MacOS (zsh)
+
+**You can also install Node Using the macOS installer available from the Node.js [website](https://nodejs.org/en/download/)**
+
+
+# Prerequisites 
+
+Make sure you have command line tools installed by typing `git` into the terminal.
+
+~~~bash
+git
+~~~
+
+This will prompt you to install command\-line tools when the command fails to work.
+
+This only applies to people wanting to upgrade from bash to zsh.
+
+To see what shell your terminal currently is in, from the command line type in:
+
+~~~bash
+echo $0
+~~~
+
+If you’re in the `bash` shell, you’ll see printed out:
+
+~~~bash
+\-bash
+~~~
+
+If you’re in the `zsh` shell, you’ll see printed out:
+~~~bash
+\-zsh
+~~~
+# Switch Your Default Shell To `zsh`
+
+To switch your shell to the `zsh` shell, you may enter:
+
+~~~bash
+chsh \-s /bin/zsh
+~~~
+
+To switch back to the bash shell, you may enter:
+
+~~~bash
+chsh \-s /bin/bash
+~~~
+
+
+# Follow the instructions :
+
+Make a folder under your home directory called dev , something like this:
+
+
+~~~zsh
+mkdir ~/.antigen && cd ~/.antigen
+~~~
+
+Next, Add package manager for zsh called antigen:
+
+
+~~~zsh
+curl -L git.io/antigen > antigen.zsh
+~~~
+
+Now we’re going to configure our .zshrc file:
+
+~~~zsh
+nano ~/.zshrc
+~~~
+
+
+This will open the .zshrc file in nano.
+Copy and paste the following into your .zshrc file:
+
+~~~zsh
+
+source ~/.antigen/antigen.zsh
+# This loads the oh-my-zsh's library.
+antigen use oh-my-zsh
+# Bundles from the default repo (robbyrussell's oh-my-zsh).
+antigen bundle lukechilds/zsh-nvm
+antigen bundle git
+antigen bundle heroku
```

Comment:
I never asked for the instructions to include these tools.

Reply:
replacing .bashrc and .bash_profile with the relevant zsh equivalents one may run into permissions issues, doing it this way one won’t have permissions issues when using NPM packages.
Thing listed will be required in later.